src/App.js

```diff
@@ -1,80 +1,38 @@
 import React, { useState } from 'react';
 import { BrowserRouter as Router, Switch, Route, Link } from 'react-router-dom';
 import './App.css';
-
+import ListShopping from './components/ListShopping';
+import Home from './components/Home/home';
 import TokenList from './components/List/tokenList';
-import Product from './components/Product/Product';
-import ListProduct from './components/Product/ListProduct';
 
 export default function App() {
   const [isActive, setIsActive] = useState('List');
 
   return (
     <Router>
-      <div className="tokenButton">
-        <button
-          className={`botton ${isActive === 'tokenList' && 'bold-button'}`}
-          onClick={() => setIsActive('tokenList')}
-        >
-          <Link to="/token-list" className="no-decoration">
-            Create List
-          </Link>
-        </button>
-      </div>
       <div className="body">
         <div className="Buttons">
           <button
-            className={`botton ${isActive === 'list' && 'bold-button'}`}
-            onClick={() => setIsActive('list')}
+            className={`botton ${isActive === 'home' && 'bold-button'}`}
+            onClick={() => setIsActive('home')}
           >
-            <Link to="/list" className="no-decoration">
-              List
-            </Link>
-          </button>
-          <button
-            className={`botton ${isActive === 'add' && 'bold-button'}`}
-            onClick={() => setIsActive('add')}
-          >
-            <Link to="/add-product" className="no-decoration">
-              Add product
+            <Link to="home" className="no-decoration">
+              Home
             </Link>
           </button>
         </div>
       </div>
       <Switch>
-        <Route path="/token-list">
-          <TokenList />
+        <Route path="/home" exact>
+          <Home />
```

Comment:
@Lizhtml @Yesika-Castillo - this should be either `/` or we should redirect the users to `/home` the moment they open the browser.

Using `/` is way easier by the way 😅

Reply:
I did not understand jejejeje 😅

Reply:
@Lizhtml sorry I was not clear.

What I meant is that we shouldn't have a `/home` route, we should have a `/` route instead. Like what we saw in our pairing session yesterday.

The goal is that everytime the user opens the app, it will see the form to join a list.